### PR TITLE
chapter_13

### PR DESCRIPTION
**Pull request by lrmi0808**
Ready

### CODE REVIEW

**Comment by lrmi0808** [comment]
Ready

File: app/assets/stylesheets/custom.scss [hunk 1]
```diff
@@ -125,3 +125,60 @@ input.btn.btn-primary {
...
+    display: block;
+    margin-left: 60px;
+  }
+}
+
+input#micropost_image{
+  margin-top: 10px;
+}
+
+
```
Comment (anlvq-1058) [inline app/assets/stylesheets/custom.scss:184]: remove 2 blank line, please

File: test/models/micropost_test.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+require "test_helper"
+
```
Comment (anlvq-1058) [inline test/models/micropost_test.rb:2]: delete this file, please

File: test/fixtures/microposts.yml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Read about fixtures at https://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
```
Comment (anlvq-1058) [inline test/fixtures/microposts.yml:3]: delete this file, please

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by lrmi0808** [comment]
Ready

File: app/controllers/microposts_controller.rb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+  private
+
+  def micropost_params
+    params.require(:micropost).permit :content, :image
+  end
+
+  def correct_user
+    @micropost = current_user.microposts.find_by id: params[:id]
+  return if @micropost
```
Comment (anlvq-1058) [inline app/controllers/microposts_controller.rb:33]: indent please

**Review by anlvq-1058 — COMMENTED** [review]
cmt

**Comment by lrmi0808** [comment]
Ready

**Review by anlvq-1058 — COMMENTED** [review]
merged

**Pull request merged**